webapp/src/containers/modals/SubscribeModal/index.tsx

```diff
@@ -84,15 +88,6 @@ const SubscribeModal = () => {
         };
     };
 
-    // Get channel state
-    const getChannelState = () => {
```

Comment:
I thought to keep this as it could be useful when needed to call any MM API and store data in a redux state, but yes for now it will be of no use so removed it.
Also, have saved this for future use in a separate branch `mattermost_service`.